api/src/main/java/org/openmrs/module/aijar/api/deploy/bundle/UgandaEMRPatientFlagMetadataBundle.java

```diff
@@ -43,6 +44,7 @@ public void install() throws Exception {
         install(Flags.DUE_FOR_SECOND_DNA_PCR);
         install(Flags.OVERDUE_FOR_SECOND_DNA_PCR);
         install(Flags.DUE_FOR_RAPID_TEST);
+        install(Flags.OVER_DUE_FOR_THIRD_DNA_PCR);
```

Comment:
Overdue is one word. Follow the convention is in use on "Rapid Test", "First DNA PCR" and "Second DNA PCR".

Reply:
Done